### ISSUE TITLE
wok - plugins - kimchi - run_tests.sh not working

### ISSUE THREAD

**Issue by pvital** [kimchi-project/kimchi#726]
The unit tests provided by Kimchi plugin inside wok code is not working. To reproduce, follow the steps:

1. clone kimchi code 
2. move to wok branch
3. install wok and kimchi plugin (following the steps in README.md files)
4. from the kimchi tests directory, execute:

```
$ ./run_tests.sh
``` 

The following output is prompted:

```
Traceback (most recent call last):
  File "/usr/lib64/python2.7/runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/lib64/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/usr/lib64/python2.7/unittest/__main__.py", line 12, in <module>
    main(module=None)
  File "/usr/lib64/python2.7/unittest/main.py", line 94, in __init__
    self.parseArgs(argv)
  File "/usr/lib64/python2.7/unittest/main.py", line 149, in parseArgs
    self.createTests()
  File "/usr/lib64/python2.7/unittest/main.py", line 158, in createTests
    self.module)
  File "/usr/lib64/python2.7/unittest/loader.py", line 130, in loadTestsFromNames
    suites = [self.loadTestsFromName(name, module) for name in names]
  File "/usr/lib64/python2.7/unittest/loader.py", line 91, in loadTestsFromName
    module = __import__('.'.join(parts_copy))
  File "test_model.py", line 30, in <module>
    import wok.objectstore
ImportError: No module named wok.objectstore
```

**Comment by pvital** [comment]
Patch submitted to community mailing list.
http://lists.ovirt.org/pipermail/kimchi-devel/2015-September/011345.html

**Comment by alinefm** [comment]
Fixed by 1884b4dc
